receiver/githubreceiver/README.md

```diff
@@ -89,7 +85,9 @@ The available scrapers are:
 >   change when rebases occur, recreating the commits with new timestamps.
 
 
+<!-- TODO: Combine this documentation once the scraper code is restructured due scope change -->
+
 For additional context on GitHub scraper limitations and inner workings please
-see the [GitHub Scraper README][ghsread].
+see the [Scraping README][ghsread].
 
-[ghsread]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/gitproviderreceiver/internal/scraper/githubscraper/README.md#github-limitations
+[ghsread]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/githubreceiver/internal/scraper/githubscraper/README.md#github-limitations
```

Comment:
I think something like this would also fix it and it (arguably) makes more sense to do this for links internal to this repository
```suggestion
[ghsread]: internal/scraper/githubscraper/README.md#github-limitations
```